mirage/utils/utils.py

```diff
@@ -424,6 +424,22 @@ def expand_environment_variable(variable_name, offline=False):
     return variable_directory
 
 
+def flatten_nested_list(nested_list):
```

Comment:
Any reason you could convert the nested list to a numpy array and use `.ravel()`?

Reply:
There's no guarantee that all the sub-lists within the list have the same length, so converting to a numpy array doesn't work.
e.g. the nested list could look like: [[1,2,,3],[4,5],[6],[7,8,9]]